wasm2c/README.md

```diff
@@ -187,20 +182,6 @@ from stack-size exhaustion). This defaults to 500:
 #endif
 ```
 
-Next we can specify a module prefix. This is useful if you are using multiple
-modules that may use the same name as an export. Since we only have one module
-here, it's fine to use the default which is an empty prefix:
-
-```c
-#ifndef WASM_RT_MODULE_PREFIX
```

Comment:
The "Exported symbols" section of wasm2c/README.md (lines 318-337 in the revised file) also needs to be revised now. (I guess GitHub doesn't let me leave a comment down there because there are no changes...)